eng/build.sh

```diff
@@ -38,6 +38,10 @@ usage()
   echo "                                  [Default: Debug]"
   echo "  -runtimeFlavor (-rf)            Runtime flavor: CoreCLR or Mono."
   echo "                                  [Default: CoreCLR]"
+  echo "  -primaryRuntime                 Primary Runtime: CoreCLR or Mono. Unlike -runtimeFlavor, this produces a .NET-like"
```

Comment:
Don't we need a corresponding change in build.ps1 for Windows?

Reply:
Yes, we do. I need to learn Windows/Powershell :'(

Reply:
I don't have access to a Windows environment to fully test my changes, but I have used the powershell docker container image and `Set-PSDebug -Trace 1` to confirm that 

    ./eng/build.ps1 -primaryRuntime Mono -v d

Results in:

    DEBUG:    1+  >>>> & "/runtime/eng/common/build.ps1" /p:TargetArchitecture=x64 -restore -build /p:PrimaryRuntimeFlavor=mono -verbosity d -configuration Debug